src/hatti/ona/forms.cljs

```diff
@@ -211,12 +223,26 @@
   (and (string? appearance-value)
        (re-matches #"^search\(.*\)$" appearance-value)))
 
+(defn media-files-upload-error-component
+  "Displays tip question and tooltip msg for failed media files uploads"
+  [answer & {:keys [help-url error-message record-modal?]}]
+  (if record-modal?
+    [:span.image-name answer
+     [:a.tooltip {:href help-url :target "_blank"}
+      [:span.tip-info error-message]
+      [:span.tip-question "?"]]]
+    (format "<span class='image-name'> %s </span>
+          <a class='tooltip' href=%s target='_blank'>
+          <span class='tip-info'>%s</span>
+          <span class='tip-question'>?</span></a>"
+            answer help-url error-message)))
```

Comment:
Any reason why the else block has to use `format` function instead of the sablono syntax used in if block?

Reply:
The sablono syntax is for the `record view` after double clicking a record on the `table view` because the `format` function does not work in this case while sablono syntax in tern does not work when viewing records on table rows thus the `format` function.

Reply:
So when `record-modal?` is true meaning a record is being viewed then the sablono syntax is applied to display the tooltip and if it's false then a format function is applied to display the tooltip on the table view side.